### PR TITLE
Removed node from call to get_executable path in ros2 run command,

### PR DESCRIPTION
**Pull request by allenh1**
see ros2/ros2cli#12.

### CODE REVIEW

File: ros2run/ros2run/command/run.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def main(self, *, parser, args):
         with NodeStrategy(args) as node:
             try:
                 path = get_executable_path(
-                    node=node, package_name=args.package_name,
+                    package_name=args.package_name,
```
Comment (wjwwood) [inline ros2run/ros2run/command/run.py]: If you're removing node here, doesn't the `with NodeStrategy(args) as node:` above this become unused?

**Comment by dirk-thomas** [comment]
The whole `with` block and the variable `node` need to go away.

**Comment by allenh1** [comment]
@wjwwood good point, fixed to reflect the changes.

File: ros2run/ros2run/command/run.py [hunk 2]
```diff
@@ -42,19 +42,18 @@ def add_arguments(self, parser, cli_name):
...
+                executable_name=args.executable_name)
+        except PackageNotFound:
+            raise RuntimeError(
+                "Package '{args.package_name}' not found"
+                .format_map(locals()))
+        except MultipleExecutables as e:
+            msg = 'Multiple executables found:'
+            for p in e.paths:
+                msg += '\n- {p}'.format_map(locals())
                 raise RuntimeError(msg)
```
Comment (allenh1) [inline ros2run/ros2run/command/run.py]: forgot to indent this line

Comment (dirk-thomas) [inline ros2run/ros2run/command/run.py]: Now this line is indented wrongly. It was on the same level as `for` before.

**Comment by mikaelarguedas** [comment]
* Linux packaging [Build Status](http://ci.ros2.org/job/ci_packaging_linux/24/)

**Comment by allenh1** [comment]
@dirk-thomas fixed -- should I rerun the CI?

**Pull request merged**